src/services/lokapiService.ts

```diff
@@ -12,11 +12,34 @@ class LokAPI extends LokAPIBrowserAbstract {
     cyclos
   }
 
-  requestLocalPassword = async function (state: string) {
+  rememberedPassword = { pwd: '', inputTime: 0 }
+
+  requestLocalPassword = async (state: string) => {
     let text
+
+    // Lets see if we have a (still valid) password in there...
+    // ... that was used less than 30secs ago
+    // If it's the case, we will provide it to the lokapi right away
+    if (this.rememberedPassword.pwd.length > 0
+      && this.rememberedPassword.inputTime + 30000 > Date.now()
+      && state === 'firstTry'
```

Comment:
It would be better not to test several time for the same test.

Reply:
@vaab allright, rebased on the current main branch, and tested to really check if the remember password feature works as intended, and it does.
This PR is stand-alone.